src/PMGDQueryHandler.h

```diff
@@ -119,6 +119,7 @@ namespace VDMS {
 
     public:
         static void init();
+        static void destroy();
```

Comment:
Why isn't this a destructor?

Reply:
just pairs up with an init

Reply:
Okay, I can see that. But init is needed (I assume) because the information required to perform initialization isn't available at construction time. The same concern doesn't apply to using a destructor, and I think performing this cleanup work in the destructor would be preferable.

Reply:
As you pointed out, the object is a static member, so the destructor isn't called at the time when this destroy function is called.